app/controllers/account_reset/recovery_options_controller.rb

```diff
@@ -0,0 +1,30 @@
+module AccountReset
+  class RecoveryOptionsController < ApplicationController
+    include TwoFactorAuthenticatable
+
+    before_action :confirm_two_factor_enabled
+
+    def show
+      recovery_options_visit
+    end
+
+    def cancel
+      cancel_account_reset_recovery_options
+      redirect_to login_two_factor_options_url
+    end
+
+    private
+
+    def confirm_two_factor_enabled
+      return if MfaPolicy.new(current_user).two_factor_enabled?
+    end
```

Comment:
Should there be logic here for what to do if the method's early `return` isn't triggered?